package.json

```diff
@@ -6,6 +6,9 @@
   "repository": "muut/riotjs",
   "author": "Muut, Inc. and other contributors",
   "license": "MIT",
+  "scripts": {
+    "test": "make test"
```

Comment:
We need to decide which tool we want to use,all the terminal command were inside the make file so I keep using it.If we want to use npm it's better to move all the make commands in the package.json (I won't do that). In any case travis-ci will trigger `npm run test` by default that's the reason why it's there